### PR TITLE
Prevent deletion of core settings

### PR DESCRIPTION
**Pull request by Tropicalista**
Not sure if this will work on ACF or Railo.

### CODE REVIEW

File: modules/contentbox-admin/views/settings/rawSettingsTable.cfm [hunk 1]
```diff
@@ -53,7 +53,7 @@
 					</a>
 					
 					<!--- Delete Command --->
-					<a 	class="btn btn-sm btn-danger confirmIt" 
+					<a 	class="btn btn-sm btn-danger #setting.getIsCore() ? '' : 'confirmIt'#"
```
Comment (lmajano) [inline modules/contentbox-admin/views/settings/rawSettingsTable.cfm]: does it make sense to add this @Tropicalista if you already have an if statement below on line 58

**Comment by Tropicalista** [comment]
The if statement does not prevent to click on disabled button, probably due to the confirmIt class presence that trigger the confirmation dialog. 

I have tested on latest chrome and core settings can be deleted also if the buttons are disabled.

Please could you check this?

File: modules/contentbox-admin/views/settings/rawSettingsTable.cfm (hunk 1 above)
Comment (lmajano) [inline modules/contentbox-admin/views/settings/rawSettingsTable.cfm]: Shouldn't this be in hashes?

Comment (Tropicalista) [inline modules/contentbox-admin/views/settings/rawSettingsTable.cfm]: In lucee you could write ternary operator in hashes and get executed, not sure in ACF.

**Comment by lmajano** [comment]
I think the fixed you did before fixed this.

**Pull request closed without merging**